client/src/components/Applications/AppForm/AppForm.jsx

```diff
@@ -44,6 +44,7 @@ const appShape = PropTypes.shape({
     coresPerNode: PropTypes.number,
     maxMinutes: PropTypes.number,
     tags: PropTypes.arrayOf(PropTypes.string),
+    queueFilter: PropTypes.arrayOf(PropTypes.string),
```

Comment:
I think we can drop these changes on lines 47 and 272, as these don't correspond to default fields on the form

Reply:
Yep this change addresses Chandra's comment